djconnectwise/sync.py

```diff
@@ -919,6 +924,7 @@ def parent_object_ids(self):
 class TeamSynchronizer(BoardFilterMixin, BoardChildSynchronizer):
     client_class = api.ServiceAPIClient
     model_class = models.TeamTracker
+    m2m_changed = False
```

Comment:
With the other changes I mentioned, I don't this should be necessary anymore.

Reply:
Thanks for your review. I thought the same thing at first, but I changed my mind and thought it could be okay. But after getting your review, it is applied for now.

However, there is a little difference: 
Before line 288, `elif self._is_instance_changed(instance):`, I prefer assigning all data to instance (i.e. not in `_is_instance_changed`, but in `_process_members`), so the member variable is still there. 

Please review that, and let me know your thought. Thanks a lot.

Reply:
As discussed it's not ideal but we're going to keep the check if the m2m has changed in `_assign_field_data` since putting it in `_is_instance_changed` won't work since by that point the API data has already been set on the instance. 


Thanks for your work on this Daniel. I changed things a little bit just to make to simplify and make sure the method that checks if the m2m has changed only does that one thing.

Reply:
I'll merge shortly